### ISSUE TITLE
Python Mode has disappeared from the contribution manager window

### ISSUE THREAD

**Issue by processing-bot** [processing/processing4#273]
*Created by: tabreturn*

Python Mode has disappeared from the Contribution Manager window list.

Not sure where to post this (in Processing or Processing4). Of course, Python Mode is for Processing 3, but I suspect that source code isn't the issue.

More info: https://github.com/processing/processing/issues/6226

**Comment by processing-bot** [comment]
*Created by: tabreturn*

It seems Python Mode has is missing in this list: http://download.processing.org/contribs

I assume this is causing the issue? Not sure how the contents of this file are managed?

**Comment by processing-bot** [comment]
*Created by: tabreturn*

I did some digging and found this: https://github.com/processing/processing-website/blob/main/content/contributions/Python_Mode_for_Processing_3.json

I'm assuming this JSON generates the http://download.processing.org/contribs? Looks like the py.processing.org certificate has expired for py.processing.org. Is this causing the problem?

**Comment by processing-bot** [comment]
*Created by: HarryJain*

Yes I noticed this too. The website is also getting blocked on both Safari and Chrome by default!

**Comment by processing-bot** [comment]
*Created by: tabreturn*

I've [reached out](https://github.com/jdf/processing-py-site/issues/204) to the person who controls the server. I'm assuming this isn't a complicated issue to resolve (SSL certificate renewal), so hopefully things are resolved soon.

**Comment by processing-bot** [comment]
*Created by: tabreturn*

Python Mode has reappeared in the Contribution Manager window 😄

**Comment by processing-bot** [comment]
*Created by: github-actions[bot]*

This issue has been automatically locked. To avoid confusion with reports that have already been resolved, closed issues are automatically locked 30 days after the last comment. Please open a new issue for related bugs.
